src/client/repository/RoomsRepository.js

```diff
@@ -1,20 +1,22 @@
 /**
  * RoomsRepository
  *
- * @param {SocketCLient} client
+ * @param {SocketClient} client
  */
 function RoomsRepository(client)
 {
     EventEmitter.call(this);
 
-    this.client = client;
-    this.rooms  = new Collection([], 'name');
+    this.client       = client;
+    this.rooms        = new Collection([], 'name');
+    this.nbPlayers    = 0;
```

Comment:
Not a big fan of abbreviations in variable names: maybe rename to `totalPlayers` or `totalClients`?